whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapServiceTestIntegration.java

```diff
@@ -3336,26 +3324,25 @@ public void get_ipv6_top_not_found(){
     public void get_ipv6_top_found_if_assignment(){
         loadIpv6RelationTreeExample();
 
-        databaseHelper.updateObject("" +
-                "inet6num:       2000::/3\n" +
+        databaseHelper.addObject("" +
```

Comment:
I checked

rdapLookupService.lookupObject

and I see that we are validating RdapObjectMapper.isIANABlock(resultObject)
We never return ::/0 or 0/0 in lookups 
so I created a parent for 2000::/3 to avoid to hit ::/0